tsconfig.json

```diff
@@ -32,7 +32,9 @@
       "@testhelpers": ["./src/__tests__/helpers"],
       "@utils/*": ["./src/utils/*"],
       "@domain/*": ["./src/domain/*"],
-      "@hooks/*": ["./src/hooks/*"]
+      "@hooks/*": ["./src/hooks/*"],
+      "@backend-api/templates/*": ["./lambdas/backend-api/src/templates/*"],
```

Comment:
This is here to appease the linting lords. It shouldn't really be here. It should be getting resolved as part of https://nhsd-jira.digital.nhs.uk/browse/CCM-7005